spec/api/password_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'Password changing endpoint test' do
+  describe 'POST /api/password/update' do
+    let(:url) { '/api/password/update' }
+    let!(:password0) { 'testpassword111' }
+    let!(:password1) { 'testpassword123' }
+    let(:params0) do
+      {
+        old_password: password0,
+        new_password: password1
```

Comment:
don't use old_password, use doorkeeper shared context

Reply:
it was techincal specs :)

Reply:
ok